tests/components/rainforest_eagle/test_config_flow.py

```diff
@@ -101,16 +102,26 @@ async def test_import(hass: HomeAssistant) -> None:
         "homeassistant.components.rainforest_eagle.async_setup_entry",
         return_value=True,
     ) as mock_setup_entry:
-        result = await hass.config_entries.flow.async_init(
-            DOMAIN,
-            data={CONF_CLOUD_ID: "abcdef", CONF_INSTALL_CODE: "123456"},
-            context={"source": config_entries.SOURCE_IMPORT},
+        await async_setup_component(
```

Comment:
This kind of test, end to end, is normally done in the `test_init.py` module or in the platform tests.

Reply:
Well we don't test the init here for the domain, we test the init of sensor setting up platform -> importing the config flow.

Reply:
Right, that's why I mentioned platform tests. From platform, import, flow, integration setup, platform setup, entity state.

Reply:
moved to init.